lib/liquid/standardfilters.rb

```diff
@@ -391,6 +391,17 @@ def round(input, n = 0)
       raise Liquid::FloatDomainError, e.message
     end
 
+    # Defaults are passed as nil so systems can easily override
+    def format_number(input, options = {})
+      options = {} unless options.is_a?(Hash)
+      precision = options['precision'] || 2
+      delimiter = options['delimiter'] || " ".freeze
+      separator = options['separator'] || ".".freeze
+      return input if (prec = Utils.to_number(precision).to_i) < 0
```

Comment:
Would move `prec = Utils.to_number(precision).to_i` to its own line for readability.